.drone.star

```diff
@@ -348,12 +348,8 @@ def getOCISTestSteps(server, db):
 		installNPM() +
 		buildPhoenix() +
 		installCore(server, db) +
-		buildKonnectd() +
-		buildReva() +
-		buildOcisPhoenix() +
-		revaService() +
```

Comment:
You can remove the function definition for `buildReva()` and `revaService()` now that we don't need them anymore